pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -630,7 +635,19 @@ spec:
       - {{ $v | quote }}
       {{- end }}
       {{- end }}
+    {{- else -}}
+    {{- $poolUids := keys .ListItems.cvolPoolList.pools | randomize }}
+    {{- $replicaCount := .Config.ReplicaCount.value | int64 -}}
```

Comment:
code from here seems to be duplicate.. can we make this as common to both (if case and else case)?

Reply:
will try with ternary

Reply:
done